scripts/manage_subscription.py

```diff
@@ -56,9 +56,8 @@ def pay_subscription(account, network, domain, subscription_contract, encryptor_
     check_plugins()
     click.echo(f"Connected to {network.name} network.")
     transactor = Transactor(account=account)
-    subscription_contract = registry.get_contract(
-        contract_name=subscription_contract, domain=domain
-    )
+    subscription_contract = Contract(address=subscription_contract)
```

Comment:
Similar to the comment above. There's no associated ABI, as opposed in the original code that had access to the registry. Perhaps this can work in certain local configurations, if ape has prior knowledge about deployed ABIs for a particular address, but we can't make this assumption.

Reply:
See comment above.